### DISCUSSION TITLE
extends orering  in mixed flat & flatcompat config?

### DISCUSSION THREAD

**Discussion by pgnd** [eslint/eslint#17089] [Q&A] [answered]
i'm moving a project's config to eslint's v9x flat config

flatcompat backwards compatibility 

	https://eslint.org/blog/2022/08/new-config-system-part-2/#backwards-compatibility-utility

is still needed

when mixing flat & flatcompat config -- e.g., mv'ing 'prettier' to flat -- how is extends order preserved/applied?

in the following edit, is 'prettier' *still* applied after 'airbnb-base' & 'plugin:promise/recommended' which are in FlatCompat config?

```
eslint.config.mjs
	import { FlatCompat } from "@eslint/eslintrc";
+	import prettier from "eslint-config-prettier";

	const compat = new FlatCompat({
		baseDirectory: __dirname,
	});

	export default [
	compat.extends(
		"airbnb-base",
		"plugin:promise/recommended",
-		"prettier"
	),
+	prettier,
	compat.plugins("prettier"),
	{
		files: ["**/*.js", "**/*.cjs"],
		parser: espree,
		parserOptions: {
			ecmaVersion: 2023,
			sourceType: 'module',
			ecmaFeatures: {
				jsx: true,
			},
		},
	}
];
```

**Comment by nzakas** [comment, marked as answer]
`FlatCompat#extends()` works the same way as `extends` in eslintrc so configs are applied from first to last.
